lib/v2/manhuagui/subscribe.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const { parseRelativeDate } = require('@/utils/parse-date');
+const cheerio = require('cheerio');
+const config = require('@/config').value;
+const web_url = 'https://www.manhuagui.com/user/book/shelf/1';
+const cookie = config.manhuagui.cookie;
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: web_url,
+        headers: {
+            Referer: 'https://www.manhuagui.com/',
+            Cookie: cookie,
+        },
+    });
```

Comment:
got request can be simplified to 
```suggestion
    const response = await got(web_url, {
        headers: {
            cookie,
        },
    });
```